app/src/main/res/layout/activity_search_results.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
What's the use of CoordinatorLayout here? If you are not using any of its methods then it can be replaced with Linear layout right?

Reply:
I used same as before due to scrolling behavior for the event detail fragment - 
``` java
if (scrollY > rootView.eventName.height + rootView.logo.height) {
                    /*Toolbar title set to name of Event if scrolled more than
                    combined height of eventImage and eventName views*/
                    activity?.supportActionBar?.title = eventShare.name
                } else {
                    // Toolbar title set to an empty string
                    activity?.supportActionBar?.title = " "
                }
```